<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>新生信息导入工具</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>96</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>在校生文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>61</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>新生文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freshFileName">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>64</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输入文件名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentFileName">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>99</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输入文件名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>61</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studentButton">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>96</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>691</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>194</y>
      <width>691</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="freshEdit"/>
     </item>
     <item>
      <widget class="QTextEdit" name="studentEdit"/>
     </item>
     <item>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
